src/main/java/software/amazon/awssdk/crt/http/Http2ClientConnection.java

```diff
@@ -183,7 +184,7 @@ public void updateConnectionWindow(long incrementSize) {
      *         the user thread making this request when it's done.
      */
     @Override
-    public Http2Stream makeRequest(HttpRequest request, HttpStreamResponseHandler streamHandler)
+    public Http2Stream makeRequest(HttpRequestBase request, HttpStreamResponseHandler streamHandler)
```

Comment:
Take `Http2Request` here?

Reply:
That's an override. The native code has different function for `http2ClientConnectionMakeRequest` vs `httpClientConnectionMakeRequest`, which makes sure if you makes a request from a Http2Connection, Http2Stream will be returned.